Jenkinsfile

```diff
@@ -51,11 +51,11 @@ node('docker') {
                         k3d.startK3d()
                     }
 
-                    stage('Deploy minio') {
+                    stage('Deploy minio and loki') {
                         withCredentials([[$class: 'UsernamePasswordMultiBinding', credentialsId: 'harborhelmchartpush', usernameVariable: 'HARBOR_USERNAME', passwordVariable: 'HARBOR_PASSWORD']]) {
                             k3d.helm("registry login ${registryUrl} --username '${HARBOR_USERNAME}' --password '${HARBOR_PASSWORD}'")
-                            k3d.helm("install k8s-minio oci://${registryUrl}/${registryNamespace}/k8s-minio --version 2023.9.23-3")
-                            k3d.helm("install k8s-loki oci://${registryUrl}/${registryNamespace}/k8s-loki --version 2.9.1-3")
+                            k3d.helm("install k8s-minio oci://${registryUrl}/${registryNamespace}/k8s-minio --version 2023.9.23-6")
+                            k3d.helm("install k8s-loki oci://${registryUrl}/${registryNamespace}/k8s-loki --version 3.1.1-1")
```

Comment:
Wouldn't we want to avoid an accidental release with this error in the future by installing the components using the component operator like this? https://github.com/cloudogu/k8s-ces-control/blob/develop/Jenkinsfile#L88

Reply:
This code still deploys the component with a regular `helm install`. We should create a separate user story for this.